src/App/Home/constants.js

```diff
@@ -4,11 +4,16 @@ import CodeTags from "@components/CodeTags";
 const basename = process.env.basename;
 
 export const changeLogs = [
+    {
+        version: "10.6.0",
+        title: "A brand new Utilites page 😱",
```

Comment:
typo: 
```suggestion
        title: "A brand new Utilities page 😱",
```
such a small thing I'm not even sure it's worth fixing it 🤫